File: src/test/resources/oracle/commits/rm-action-test-8187d840c38c0a1eaf8617c1290f2782084dd059/CustomerProfile.java
public class CustomerProfile {
    private String fullName;
    private String street;
    private String city;
    private String postalCode;
    public int loyaltyPoints;

    public CustomerProfile(String fullName, String street, String city, String postalCode, int loyaltyPoints) {
        this.fullName = fullName;
        this.street = street;
        this.city = city;
        this.postalCode = postalCode;
        this.loyaltyPoints = loyaltyPoints;
    }

    public String mailingLabel() {
        return fullName + "\n" + street + "\n" + city + " " + postalCode;
    }

    public boolean canReceivePromotion(boolean optedIn, boolean bouncedEmail, int purchaseCount) {
        if (optedIn && !bouncedEmail && purchaseCount > 0) {
            return true;
        }
        return false;
    }

    public void addPurchase(int amountCents) {
        int points = amountCents / 100;
        loyaltyPoints += points;
    }
}

File: src/test/resources/oracle/commits/rm-action-test-0ce4051977047c6311f397055a07218a7318d87c/python/customer_profile.py
class CustomerProfile:
    def __init__(self, full_name, street, city, postal_code, loyalty_points):
        self._full_name = full_name
        self._street = street
        self._city = city
        self._postal_code = postal_code
        self.loyalty_points = loyalty_points

    def mailing_label(self):
        return f"{self._full_name}\n{self._street}\n{self._city} {self._postal_code}"

    def can_receive_promotion(self, opted_in, bounced_email, purchase_count):
        if opted_in and not bounced_email and purchase_count > 0:
            return True
        return False

    def add_purchase(self, amount_cents):
        points = amount_cents // 100
        self.loyalty_points += points
